schema.json

```diff
@@ -6,7 +6,7 @@
     "of the License at https://www.gnu.org/licenses/agpl-3.0.en.html"
   ],
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://schemas.s1seven.com/en10168-schemas/v0.2.1/schema.json",
+  "$id": "https://schemas.s1seven.com/en10168-schemas/v0.3.0/schema.json",
```

Comment:
there is no need to manually set the version.
When the release will be done a script is called to bump versions of all files that are versioned (.json, .hbs, .pdf).
See the `release-it` config in `package.json`

Reply:
ok